src/test/kotlin/org/gradle/script/lang/kotlin/accessors/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -10,10 +10,46 @@ import org.junit.Assert.assertFalse
 import org.junit.Test
 
 import java.io.File
+import java.nio.charset.StandardCharsets.UTF_8
 
 class ProjectSchemaAccessorsIntegrationTest : AbstractIntegrationTest() {
 
     @Test
+    fun `can configure deferred configurable extension`() {
+        withBuildScript("""
+
+            import org.gradle.api.publish.maven.MavenPublication
+
+            plugins {
+                `java-library`
+                `maven-publish`
+            }
+
+            dependencies {
+                "api"("com.google.guava:guava:21.0")
+            }
+
+            publishing {
+                publications.create<MavenPublication>("mavenJavaLibrary") {
+                    from(components["java"])
+                }
+            }
+
+            dependencies {
+                "api"("org.apache.commons:commons-lang3:3.5")
+            }
+
+        """.replaceIndent())
+
+        withAutomaticAccessors()
+        build("generatePom")
+
+        val pom = existing("build/publications/mavenJavaLibrary/pom-default.xml").readText(UTF_8)
```

Comment:
`UTF_8` is already the [default](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/read-text.html).